integration-libs/opf/base/components/opf-quick-buy/apple-pay/apple-pay.component.ts

```diff
@@ -73,7 +78,7 @@ export class ApplePayComponent implements OnInit, OnDestroy {
           this.applePayService.start(
             product as Product,
             this.itemCounterService.getCounter(),
-            this.activeConfiguration.acquirerCountryCode
+            this.applePayDigitalWallet?.countryCode as string
```

Comment:
'as string' is legitimate as line 61 proves this.applePayDigitalWallet?.countryCode cannot be undefined.